tests/oeedger8r/enc/testdeepcopy.cpp

```diff
@@ -45,7 +45,7 @@ void deepcopy_count(CountStruct* s)
     OE_TEST(s->size == 64);
     for (size_t i = 0; i < 3; ++i)
         OE_TEST(s->ptr[i] == data[i]);
-    OE_TEST_IGNORE(oe_is_within_enclave(s->ptr, 3 * sizeof(uint64_t)));
+    OE_TEST(oe_is_within_enclave(s->ptr, 3 * sizeof(uint64_t)));
```

Comment:
Notable missing test is if `s == nullptr`, then the generated code will be incorrect, as it does not check `s && s->ptr` but only `s->ptr` (for now, this is obviously a TODO).